examples/advanced/variables.tf

```diff
@@ -27,3 +27,13 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "kms_key_crn" {
+  type        = string
+  description = "The root key CRN of a Key Management Services like Key Protect or Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. Only used if `kms_encryption_enabled` is set to true."
```

Comment:
`kms_encryption_enabled` is not a variable exposed in the example, so remove it from the description here